src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -92,6 +92,7 @@ public static class BuiltinNodeCategories
         public const string REVIT_PARAMETERS = "Revit.Parameters";
         public const string REVIT_BAKE = "Revit.Bake";
         public const string REVIT_API = "Revit.API";
+        public const string REVIT_ELEMENTS_DIVIDEDPATH_ACTION = "Revit.Elements.DividedPath.Actions";
```

Comment:
Hi @aosyatnik, Revit specific things have no place in `DynamoCore`, please move all these Revit stuff out of this repository, and add them to `DynamoDS/DynamoRevit:Revit2014` branch (then later create two other PRs for both `Revit2015` and `Revit2016` branches).

While you're doing this, can you also remove those strings that are no longer used? Like when I casually searched, I see that `LOGIC_MATH_ROUNDING` is not used anywhere. Only a few under `Core` is used. Thanks!